### ISSUE TITLE
Custom emoji shortcodes should be displayed inline if images don’t load

### ISSUE THREAD

**Issue by jennifermf** [tootsuite/mastodon#5743]
When a user has images disabled in their browser, toots should display the full short code for any emoji in the post. Currently, emoji are entirely useless for those folks.

I can mouseover an emoji to see its code. Please please please fix it so that mouseover text is part of the body of a toot when images are disabled.

![screen shot 2017-11-17 at 11 56 37 atm](https://user-images.githubusercontent.com/26920220/32964051-23b702b6-cb8f-11e7-94b0-6e6583b61865.png)
![screen shot 2017-11-17 at 11 58 24 atm](https://user-images.githubusercontent.com/26920220/32964053-23fcba72-cb8f-11e7-94b1-9c8800153e7e.png)

* * * *

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [ ] This bug happens on a [tagged release](https://github.com/tootsuite/mastodon/releases) and not on `master` (If you're a user, don't worry about this).

**Comment by ThibG** [comment]
I think this could be fixed by changing the `emojione` elements to not have `visibility: inline-block`. There might be downsides to this, but I can't think of any at the moment.

**Comment by jennifermf** [comment]
thank you!!
